nuxeo-compound-documents-web/ftest/itests.xml

```diff
@@ -0,0 +1,31 @@
+<project name="compound-documents-webdriver-tests" xmlns:nx="urn:nuxeo-build" xmlns:artifact="urn:nuxeo-artifact">
+  <taskdef resource="org/nuxeo/build/antlib.xml" uri="urn:nuxeo-build" />
+  <taskdef resource="org/nuxeo/build/artifact/antlib.xml" uri="urn:nuxeo-artifact" />
+  <taskdef resource="net/sf/antcontrib/antlib.xml" />
+
+  <property name="out.dir" value="${maven.project.build.directory}" />
+  <unzip dest="${out.dir}/" overwrite="false">
+    <artifact:resolveFile key="org.nuxeo:nuxeo-ftest::zip" />
+  </unzip>
+  <import file="${out.dir}/nuxeo-ftest.xml" />
+
+  <property name="mp.install" value="file:${out.dir}/nuxeo-compound-documents-package-${maven.project.version}.zip" />
```

Comment:
Good question indeed. I didn't specify the Web UI marketplace version, because it is marked as a dependency of the Nuxeo Compound Package and it will be installed from Connect. Do you think we should have an explicit version of Web UI to install here?

Reply:
I think this is ok, but we might need a way to install release candidates of Web UI soon (if we need something that was not published yet). In that case, we might need to add something here; or, point to connect preprod on the nuxeo.conf, which should install the latest RC?